acceptance/libs/telemetry/dummy-without-server/script

```diff
@@ -0,0 +1,5 @@
+export DATABRICKS_TELEMETRY_UPLOAD_LOGS_FILE=./out.upload.txt
```

Comment:
> acceptance/libs/telemetry/dummy-without-server/script

Could you remove libs from the path? We're not testing the library here but the whole command. So it's either should be `acceptance/cmd/telemetry/dummy-without-server/script` or `acceptance/telemetry/dummy-without-server/script`